testdata/workspaceConfigTemplate.json

```
@@ -470,6 +470,152 @@
+                    },
+                    "secretConfig": {},
+                    "id": "xxxyyyzzYjAZfPBa5yqqTvqBVqu",
+                    "name": "Des Redis Integration Test 1",
+                    "enabled": true,
+                    "workspaceId": "{{.workspaceId}}",
+                    "deleted": false,
+                    "createdAt": "2021-08-27T06:45:46.149Z",
+                    "updatedAt": "2021-08-27T06:45:46.149Z",
+                    "transformations": [],
+                    "destinationDefinition": {
+                        "config": {
+                            "destConfig": {
+                                "defaultConfig": [
+                                    "address",
+                                    "database",
+                                    "password",
+                                    "prefix",
+                                    "secure",
+                                    "skipVerify",
+                                    "caCertificate",
+                                    "clusterMode"
+                                ]
+                            },
+                            "secretKeys": [
+                                "password",
+                                "caCertificate"
+                            ],
+                            "excludeKeys": [],
+                            "includeKeys": [],
+                            "transformAt": "processor",
+                            "transformAtV1": "processor",
+                            "supportedSourceTypes": [
+                                "android",
+                                "ios",
+                                "web",
+                                "unity",
+                                "amp",
+                                "cloud",
+                                "warehouse",
+                                "reactnative",
+                                "flutter"
+                            ],
+                            "supportedMessageTypes": [
+                                "identify"
+                            ],
+                            "saveDestinationResponse": true
+                        },
+                        "configSchema": null,
+                        "responseRules": {},
+                        "id": "xxxyyyzzl28zT3nxr8SWZp6xaWT",
+                        "name": "REDIS",
+                        "displayName": "Redis",
+                        "category": null,
+                        "createdAt": "2021-02-08T15:26:17.995Z",
+                        "updatedAt": "2021-08-27T08:09:46.976Z"
+                    },
+                    "isConnectionEnabled": true,
+                    "isProcessorEnabled": true
+                },
+                {
+                    "config": {
+                        "hostName": "localhost",
+                        "port": "9092",
+                        "topic": "dev_integration_test",
```

Maybe template this one as well, so we define the topic only once:
```suggestion
                        "topic": "{{.minioTopic}}",",
```